frontend/src/ui/notifications/notifications.scss

```diff
@@ -0,0 +1,189 @@
+@import '../styles.scss';
+
+.notifications-container {
+  position: fixed;
+  top: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column-reverse;
+  padding: 3px;
+  max-width: calc((100vw - #{$container-width}) / 2);
+  z-index: 999;
+  
+  @media (max-width: $responsive-mobile-browser-width) {
+    max-width: 60vw;
+  }
+
+  .notification {
+    position: relative;
+    display: flex;
+    align-items: start;
+    width: 100%;
+    max-width: 35rem;
+    padding: 0.75*$space-standard 3*0.75*$space-standard 0.75*$space-standard 0.75*$space-standard;
+    transition: transform 0.3s ease-out;
+    opacity: 0.9;
+
+    &:hover {
+      opacity: 1;
+      .lifetime {
+        animation-play-state: paused;
+      }
+    }
+  
+    &:before, &:after {
+      content: '';
+      display: block;
+      position: absolute;
+      right: 0;
+      bottom: 0;
+      top: 0;
+      left: 0;
+      z-index: -1;
+      background: var(--color-alt-bg);
+      clip-path: polygon(0.6rem 0, 100% 0, 100% 100%, calc(100% - 0.6rem) 100%, 0 100%, 0 1.6rem);
+    }
+  
+    &:after {
+      right: -0.6px;
+      bottom: -0.6px;
+      top: -0.6px;
+      left: -0.6px;
+      z-index: -2;
+      background: var(--color-alt-border);
+    }
+
+    > * {
+      line-height: 2rem;
```

Comment:
If put 1.8 or 1.5em for example, icons and text will jump. I put 20px or 2 rem and everything aligns with the same setting, so I chose rem because if you change the main value, then both the text and icons will be adapted